subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -338,7 +339,13 @@ static bool validate_firmware(uint32_t fw_dst_address, uint32_t fw_src_address,
 		return false;
 	}
 
+	PRINT("Trying to get Firmware version\n\r");
```

Comment:
just try to build for any platform:
`west build -p -b nrf52840dk/nrf52840 tests/subsys/bootloader/bl_validation`
receive:
```
ncs/nrf/subsys/bootloader/bl_validation/bl_validation.c:342:9: warning: implicit declaration of function 'PRINT' [-Wimplicit-function-declaration]
  342 |         PRINT("Trying to get Firmware version\n\r");
      |         ^~~~~
```

Reply:
use `printk` instead

Reply:
The file generally uses logging, so I would say to use LOG_ instead.